Functions/Mock.ps1

```diff
@@ -330,6 +333,7 @@ about_Mocking
             Scope                   = $pester.CurrentTestGroup
             PesterState             = $pester
             Metadata                = $metadata
+            OriginalMetadata        = $originalMetdata
```

Comment:
Having the original `Metadata` at hand, would facilitate future operations where we may want to override the mock customizations in nested blocks